src/components/core.ts

```diff
@@ -182,6 +182,7 @@ export default class Core {
      * @type {{type: (*), data: {text: null}}}
      */
     const defaultBlockData = {
+      id: _.generateBlockId(),
```

Comment:
Id is not required param, so I think it should be removed from here. It will be generated in the `composeBlock` method